shunting-yard-replicator/src/test/java/com/expediagroup/shuntingyard/replicator/exec/external/CircusTrainConfigTest.java

```diff
@@ -96,6 +96,7 @@ public void withMultiValuePartitionFilter() {
     assertThat(replication.getReplicaTable().getDatabaseName()).isEqualTo("replicaDatabaseName");
     assertThat(replication.getReplicaTable().getTableName()).isEqualTo("replicaTableName");
     assertThat(replication.getReplicaTable().getTableLocation()).isEqualTo("replicaTableLocation");
+    assertThat(replication.getOrphanedDataStrategy()).isEqualTo(OrphanedDataStrategy.HOUSEKEEPING);
```

Comment:
Will be good to add a test when `OrphanedDataStrategy` is null.

The `CircusTrainConfig` doesn't do a null check on this argument at the moment.

Reply:
Good point, will add this in, thanks